packages/main/src/themes/sap_horizon_hcb/RadioButton-parameters.css

```diff
@@ -2,9 +2,15 @@
 
 :root {
 	--_ui5_radio_button_checked_fill: var(--sapContent_IconColor);
-	--_ui5_radio_button_checked_warning_fill: var(--sapContent_IconColor);
-	--_ui5_radio_button_checked_error_fill: var(--sapContent_IconColor);
+	--_ui5_radio_button_hover_fill_error: var(--sapField_InvalidBackground);
+	--_ui5_radio_button_hover_fill_warning: var(--sapField_WarningBackground);
+	--_ui5_radio_button_hover_fill_success: var(--sapField_SuccessBackground);
+	--_ui5_radio_button_hover_fill_information: var(--sapField_InformationBackground);
 	--_ui5_radio_button_hover_fill: var(--sapSelectedColor);
 	--_ui5_radio_button_border_width: 0.125rem;
 	--_ui5_radio_button_warning_error_border_dash: 5;
-}
\ No newline at end of file
+	--_ui5_radio_button_inner_ring_radius: 27.5%;
+	--_ui5_radio_button_inner_ring_color: var(--sapContent_Selected_ForegroundColor);
+	--_ui5_radio_button_outer_ring_width: 2px;
```

Comment:
Hi Lidiya, we use REMs over PXs (besides 1px), so could you replace 2px with 0.125rem here and below.

Reply:
px is used due to bug in Safari on zoom

Reply:
To add, we initially found this when adapting the RadioButton for Horizon in openui5.

Reply:
I did not know, then I am fine, I suggested to Lidiya to add a small comment so that it's not refactored by someone that is not aware of that.